src/components/productCard/CardHorizontal/CardHorizontal.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { useCart } from '../../../context/cart-context';
+import { WishlistButton } from '../../component_index';
+import { cardData } from '../../../data/cardData/cardData';
+
+import './CardHorizontal.css';
+export const CardHorizontal = ({
+  id,
+  imgSrc,
+  category,
+  cardHeading,
+  cardBadge,
+  cardSubHeading,
+  productDiscountedPrice,
+  productOriginalPrice,
+  wishlist,
+  quantity,
+  cart,
+}) => {
+  const { cartData, setCartData } = useCart();
+  const [decrementBtnState, setDecrementBtnState] = useState(false);
+  const [incrementBtnState, setIncrementBtnState] = useState(false);
+  const handleIncrement = () => {
+    for (const item of cartData) {
+      if (item.id === id) {
+        if (item.inStock === 0) {
+          setIncrementBtnState(true);
+          break;
+        }
+
+        setDecrementBtnState(false);
+        item.quantity += 1;
+        item.inStock -= 1;
+        item.productDiscountedPrice *= item.quantity;
+        item.productOriginalPrice *= item.quantity;
+        break;
+      }
+    }
+    setCartData([...cartData]);
+  };
+  const handleDecrement = () => {
+    for (const item of cartData) {
+      if (item.id === id) {
+        if (item.quantity === 1) {
+          setDecrementBtnState(true);
+          break;
+        }
+        setIncrementBtnState(false);
+        item.productDiscountedPrice /= item.quantity;
+        item.productOriginalPrice /= item.quantity;
+        item.quantity -= 1;
+        item.inStock += 1;
```

Comment:
You can destructure item in the beginning of for loop, so you don't have to write item. in front of everything.